### PR TITLE
Addition of Aave aTokens and update LEND token name

### PR DESCRIPTION
**Pull request by sirsoth**
Hello,

I am Pablo from Aave (previously ETHLend). We released our new protocol in mainnet last month, in our protocol when you deposit an asset, for example DAI, you receive aDAI in your wallet 1:1 pegged to the deposited asset. We would like to add our aTokens natively in metamask so our users are able to search and add the tokens easily. Big part of our users use metamask to connect to our platform.

Here we have a specific site for our aTokens:
https://aave.com/atokens
And the addresses of them are available at our documentation:
https://developers.aave.com/#retrieve-contract-instances

Our main site is available at https://aave.com and our platform at https://app.aave.com 
Additional resources:
Whitepaper:
https://github.com/aave/aave-protocol/blob/master/docs/Aave_Protocol_Whitepaper_v1_0.pdf
Risk assesment:
https://github.com/aave/aave-protocol/blob/master/docs/AAVE%20-%20Currencies%20risk%20assessment.pdf
Repository:
https://github.com/aave/aave-protocol

Let me know if any other information is required for this process. We have also updated our LEND token name to Aave, as in other different places:
https://coinmarketcap.com/currencies/aave/
https://www.coingecko.com/en/coins/aave

Best regards!
Pablo Candela.

### CODE REVIEW

**Comment by danfinlay** [comment]
While we finalize our policy on listing new sets of wrapped tokens, would you also consider using our watchAsset method? It is intended to allow new tokens to easily be added to a user’s wallet without them filling out forms or needing to apply for inclusion in any lists.

If for any reason this isn’t a perfect UX for your users, please help us make it better, so we can make a better permissionless wallet that also isn’t prone to administration and airdrop spam issues.

https://docs.metamask.io/guide/registering-your-token.html#code-free-example

**Comment by sirsoth** [comment]
Hello @danfinlay, is there any expected time for that policy to be released? Also we would like to know if its possible to go on with the addition of our atokens taking in account that multiple similar tokens as ours are already listed, as compound ctokens, pool together tokens or rdai as example.

**Comment by danfinlay** [comment]
Pointing to other tokens is a huge slippery slope, every future derivative will point to this one as evidence that theirs should be included too. Per the readme of this repository:

> This repository is effectively frozen. We recommend that developers of new tokens use EIP 747 to ask the user's permission to display your tokens in their wallet. This reduces the dangers of airdrop-based phishing, and reduces administrative overhead from managing this list.

To disincentivize pinging me on every social media platform for every new token, I am now closing this issue. Please use EIP-747 `wallet_watchAsset` to add new tokens to users' wallets until we have a process for reviewing and merging new token derivatives without creating a centralized burden on our team.

**Comment by danfinlay** [comment]
We've got the beginnings of a new framework in place, and while it isn't in place yet, we had reviewers go over this proposal and approve it. Sorry for the wait.

**Comment by danfinlay** [comment]
Tests failed because the provided addresses are not pre-checksummed.
You can get a checksummed address by copying the address off Etherscan, or using any other address checksumming tool:
https://coincodex.com/article/2078/ethereum-address-checksum-explained/

**Comment by sirsoth** [comment]
Hi @danfinlay I will handle this, do you mind if we also update the list adding the recently added atokens to Aave? So we have it up to date

**Comment by danfinlay** [comment]
We'll need to re-review, but you may.

**Comment by danfinlay** [comment]
This test failed because of too long token names. Please ensure you get tests passing with an open PR. We’ll need to verify longer token symbols don’t break our UI before accepting longer. Please get tests passing then ping me again.

**Comment by danfinlay** [comment]
Short term you may want to just remove the tokens with longer names, or abbreviate them for now.

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,2123 @@
...
+      "decimals": 18
+    },
+    "0x7D85e23014F84E6E21d5663aCD8751bEF3562352": {
+      "name": "Axion",
+      "logo": "axion.svg",
+      "erc20": true,
+      "symbol": "AXN",
+      "decimals": 18
+    }
+=======
```
Comment (xrn) [inline contract-map.json:2120]: Probably you should remove this.

Comment (sirsoth) [reply contract-map.json:2120]: thanks @xrn Im updating a few things and come back to you!

Comment (sirsoth) [reply contract-map.json]: Hi @xrn, everything should be fine now. There were duplicated assets from the commits that were lacking in my branch. Following the previous discussion we removed long name assets (uniswap market), also regarding the limitation we have one asset symbol which is stkAAVE (7 chars) but there is a restriction to 6 chars so we set it as stAAVE meanwhile, would be possible to add an exception over this asset to put his whole symbol 'stkAAVE'?

**Comment by xrn** [comment]
@sirsoth 

1. I see that you removed aave.svg and replaced it with AAVE.svg - in this can I ask you to remove aave.svg from repo in this PR
2. Can I ask also to remove all PNG files from this PR which are not used and are visible https://github.com/MetaMask/contract-metadata/pull/460/files here?

**Comment by sirsoth** [comment]
sure, I will remove the unused/duplicated, sorry

**Comment by sirsoth** [comment]
I will update aSUSD icon, @xrn 

what about the possibility of allowing stkAAVE symbol (7 chars)?

**Comment by xrn** [comment]
At this stage, I do not think so that It will be possible, but I will create an issue and connect this PR to it

**Comment by sirsoth** [comment]
Removed the duplicated/unused assets and updated aSUSD reducing its size to 34KB (hard to reduce it more)

**Comment by xrn** [comment]
@sirsoth thanks. I see still aETH.svg and aREP.svg icons but do not see them in the JSON - can we fix it?

**Comment by sirsoth** [comment]
yep, missed those, one sec

**Comment by sirsoth** [comment]
updated @xrn

**Review by xrn — APPROVED** [review]

**Comment by xrn** [comment]
@sirsoth thanks for your patience and contribution - after 1 year of waiting we have it!

**Comment by xrn** [comment]
@sirsoth are you sure that 0x05Ec93c0365baAeAbF7AefFb0972ea7ECdD39CF1 is correct aBAT address? And it should not be 0xE1BA0FB44CCb0D11b80F92f4f8Ed94CA3fF51D00 ?

**Comment by sirsoth** [comment]
Hello @xrn yes, we are actually merging all Aave V2 aTokens. As Aave protocol v2 was recently released we are merging the new V2 Atokens instead of V1 ones.

You can find all them listed at:
https://docs.aave.com/developers/deployed-contracts

And etherscan is already working in updating the new tokens information and also adding a V1/V2 label on the tokens to avoid confusion, for example it is already added in aAAVE

V1:
https://etherscan.io/address/0xba3d9687cf50fe253cd2e1cfeede1d6787344ed5

V2:
https://etherscan.io/address/0xffc97d72e13e01096502cb8eb52dee56f74dad7b

So to avoid confusion we are directly listing V2 aTokens, and skipping V1.

**Comment by sirsoth** [comment]
Thanks a lot @xrn if anything else is needed just let me know

**Pull request merged**